assets/tests/api.test.ts

```diff
@@ -43,27 +44,27 @@ describe("apiCall", () => {
   test("reloads the page if the response status is a redirect (3xx)", done => {
     mockFetch(302, { data: null })
 
-    window.location.reload = jest.fn()
+    jest.spyOn(browser, "reload").mockImplementation(() => {})
 
     apiCall({
       url: "/",
       parser: () => null,
     }).catch(() => {
-      expect(window.location.reload).toHaveBeenCalled()
+      expect(browser.reload).toHaveBeenCalled()
       done()
     })
   })
 
   test("reloads the page if the response status is forbidden (403)", done => {
     mockFetch(403, { data: null })
 
-    window.location.reload = jest.fn()
+    jest.spyOn(browser, "reload").mockImplementation(() => {})
```

Comment:
Could we just have a single spy for the whole file or describe, instead of one for each test? (We'd have to be careful that the mock call history doesn't pollute between the tests.)

Might it even be possible and worth doing to set up a mock in `tests/setup.ts` for all tests, assuming we never want the unmocked reload?